wizard/WizardMain.qml

```diff
@@ -233,6 +245,7 @@ ColumnLayout {
         appWindow.persistentSettings.auto_donations_amount = false //settings.auto_donations_amount
         appWindow.persistentSettings.restore_height = (isNaN(settings.restore_height))? 0 : settings.restore_height
         appWindow.persistentSettings.is_recovering = (settings.is_recovering === undefined)? false : settings.is_recovering
```

Comment:
Some code compares typeof(x) to undefined, this compares the value. Can a boolean be "undefined"  in javascript ?

Reply:
I'm guessing it's a fail-safe against invalid attribute access, for whenever `is_recovering` is not assigned to the object `settings` at all. [Usages](https://github.com/monero-project/monero-gui/search?q=is_recovering&unscoped_q=is_recovering) show that such situation could perhaps take place, all tough I'm not completely sure.

Kind of off-topic: The current variable scope for js is kind of a mess and I'd vouch to get everything into seperate `.js` files, with classes that encapsulate logic.